CMakeLists.txt

```diff
@@ -48,22 +48,29 @@ SET (CMAKE_VERBOSE_MAKEFILE 0) # Use 1 for debugging, 0 for release
 #
 # Project Output Paths
 #
-#MESSAGE(${PROJECT_SOURCE_DIR})
-#SET (MAINFOLDER ${PROJECT_SOURCE_DIR})
 SET (LEMONADE_DIR ${PROJECT_SOURCE_DIR})
-#MESSAGE(${LEMONADE_DIR})
-SET (EXECUTABLE_OUTPUT_PATH "${LEMONADE_DIR}/build/bin")
-SET (LIBRARY_OUTPUT_PATH "${LEMONADE_DIR}/lib")
+
+#
+# Check if LEMONADE_DIR is given
+#
+if (DEFINED INSTALLDIR_LEMONADE)
+    message("INSTALLDIR_LEMONADE set to "${INSTALLDIR_LEMONADE})
+else()
+    SET (INSTALLDIR_LEMONADE "${LEMONADE_DIR}/build/")
```

Comment:
Theres still the question how many paramter we would like to submit into the call of CMake. I think for most puposes it would be expected. But you're right. Maybe we should think about to use the [GNU-Standard] (http://www.gnu.org/prep/standards/standards.html#Managing-Releases).